cmd/kubelet/app/server.go

```diff
@@ -266,6 +265,7 @@ HTTP server: The kubelet can also listen for HTTP and respond to a simple API
 				klog.ErrorS(err, "Failed to initialize logging")
 				os.Exit(1)
 			}
+			cliflag.PrintFlags(cleanFlagSet)
```

Comment:
This is the same pattern as in all other commands which use log options and PrintFlags, for example https://github.com/kubernetes/kubernetes/blob/3b9bd229b2fafce16ba056d9c3e7f9acc3fd0bc6/cmd/kube-apiserver/app/server.go#L120-L125